cmd/backup-manager/app/clean/clean.go

```diff
@@ -67,7 +68,7 @@ func (bo *Options) cleanRemoteBackupData(bucket string, opts []string) error {
 	destBucket := util.NormalizeBucketURI(bucket)
 	args := util.ConstructArgs(constants.RcloneConfigArg, opts, "deletefile", destBucket, "")
 	output, err := exec.Command("rclone", args...).CombinedOutput()
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "doesn't exist") {
```

Comment:
does `rclone` document its exit codes?

Reply:
checking the error string is fragile, can we use the exit codes [documented here](https://rclone.org/docs/#list-of-exit-codes)?

Reply:
```log
E0710 03:41:09.430348       1 manager.go:79] clean cluster demo/basic-backup backup s3://chunzhu/backup-2020-07-09T06:31:00Z.tgz failed, err: cluster demo/basic-backup, execute rclone deletefile command failed, output: 2020/07/10 03:41:09 ERROR : Attempt 1/3 failed with 1 errors and: s3:chunzhu/backup-2020-07-09T06:31:00Z.tgz is a directory or doesn't exist
```
It sames that `rclone` dosen't document exit codes in command lines.

Reply:
> checking the error string is fragile, can we use the exit codes [documented here](https://rclone.org/docs/#list-of-exit-codes)?

nice advise! I will take a try